package ru.senina.itmo.lab8.exceptions;

public class UserAlreadyExistsException extends RuntimeException{
}
